src/lib/operations.ts

```diff
@@ -339,69 +413,74 @@ const prepareToEditHelix: PrepareToEditCallback = async ({ operation }) => {
   if (
     !('angleStart' in operation.labeledArgs) ||
     !operation.labeledArgs.angleStart
-  )
+  ) {
     return baseCommand
+  }
   const angleStart = await stringToKclExpression(
     codeManager.code.slice(
       operation.labeledArgs.angleStart.sourceRange[0],
       operation.labeledArgs.angleStart.sourceRange[1]
     )
   )
-  if (err(angleStart) || 'errors' in angleStart) return baseCommand
+  if (err(angleStart) || 'errors' in angleStart) {
+    return baseCommand
+  }
 
   // counterClockWise options boolean arg
   if (
     !('counterClockWise' in operation.labeledArgs) ||
     !operation.labeledArgs.counterClockWise
-  )
+  ) {
     return baseCommand
+  }
   const counterClockWise =
     codeManager.code.slice(
       operation.labeledArgs.counterClockWise.sourceRange[0],
       operation.labeledArgs.counterClockWise.sourceRange[1]
     ) === 'true'
 
   // radius kcl arg
-  if (!('radius' in operation.labeledArgs) || !operation.labeledArgs.radius)
+  if (!('radius' in operation.labeledArgs) || !operation.labeledArgs.radius) {
+    console.log(
+      "!('radius' in operation.labeledArgs) || !operation.labeledArgs.radius"
+    )
     return baseCommand
+  }
   const radius = await stringToKclExpression(
     codeManager.code.slice(
       operation.labeledArgs.radius.sourceRange[0],
       operation.labeledArgs.radius.sourceRange[1]
     )
   )
-  if (err(radius) || 'errors' in radius) return baseCommand
-
-  // axis options string arg
-  if (!('axis' in operation.labeledArgs) || !operation.labeledArgs.axis)
+  if (err(radius) || 'errors' in radius) {
     return baseCommand
-  const axis = codeManager.code
-    .slice(
-      operation.labeledArgs.axis.sourceRange[0],
-      operation.labeledArgs.axis.sourceRange[1]
-    )
-    .replaceAll("'", '') // TODO: fix this crap
+  }
 
   // length kcl arg
-  if (!('length' in operation.labeledArgs) || !operation.labeledArgs.length)
+  if (!('length' in operation.labeledArgs) || !operation.labeledArgs.length) {
     return baseCommand
+  }
   const length = await stringToKclExpression(
     codeManager.code.slice(
       operation.labeledArgs.length.sourceRange[0],
       operation.labeledArgs.length.sourceRange[1]
     )
   )
-  if (err(length) || 'errors' in length) return baseCommand
+  if (err(length) || 'errors' in length) {
+    return baseCommand
+  }
```

Comment:
Super not relevant here but I'd like us to adopt the https://eslint.org/docs/latest/rules/curly rule, which makes this the norm. It's so much easier to add logs (like I've had to do here) and generally maintain code in brackets vs single line or same line IMHO.